sources/us/tx/gillespie.json

```diff
@@ -21,6 +21,7 @@
             "GillespieCad.dbo.Accounts.situs_street",
             "GillespieCad.dbo.Accounts.situs_street_sufix"
         ],
-        "region": "GillespieCad.DBO.Accounts.situs_state"
+        "region": "GillespieCad.DBO.Accounts.situs_state",
+        "postcode": "gillespiecad.dbo.accounts.zip"
```

Comment:
`gillespiecad.dbo.accounts.zip` in this case refers to the owners' mailing address zip, not the physical address zip.  There is a field named `GillespieCad.dbo.Accounts.situs_zip` but it doesn't contain zip code data, only apparently errors from other fields.